Sources/ReownWalletKit/WalletKit.swift

```diff
@@ -42,11 +43,12 @@ public class WalletKit {
         metadata: AppMetadata,
         crypto: CryptoProvider,
         pushHost: String = "echo.walletconnect.com",
-        environment: APNSEnvironment = .production
+        environment: APNSEnvironment = .production,
+        pimlicoApiKey: String? = nil
```

Comment:
Do we really want to expose this as a param? Soon it will just need project ID

Reply:
but for now, there is no way around